File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/Utenlandsopphold.java
package no.nav.syfo.controller.domain.sykepengesoknad;

import java.io.Serializable;
import java.util.List;

public class Utenlandsopphold implements Serializable {
    public List<Datospenn> perioder;
    public Boolean soektOmSykepengerIPerioden;

    public Utenlandsopphold withSoektOmSykepengerIPerioden(final Boolean soektOmSykepengerIPerioden) {
        this.soektOmSykepengerIPerioden = soektOmSykepengerIPerioden;
        return this;
    }

    public Utenlandsopphold withDatospenn(List<Datospenn> datospenn) {
        this.perioder = datospenn;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykepengesoknad/Utenlandsopphold.kt
package no.nav.syfo.controller.domain.sykepengesoknad

import java.io.Serializable

class Utenlandsopphold : Serializable {
    var perioder: List<Datospenn>? = null
    var soektOmSykepengerIPerioden: Boolean? = null
    fun withSoektOmSykepengerIPerioden(soektOmSykepengerIPerioden: Boolean?): Utenlandsopphold {
        this.soektOmSykepengerIPerioden = soektOmSykepengerIPerioden
        return this
    }

    fun withDatospenn(datospenn: List<Datospenn>?): Utenlandsopphold {
        perioder = datospenn
        return this
    }
}
